src/server/req_rescq.c

```diff
@@ -589,12 +590,14 @@ req_confirmresv(struct batch_request *preq)
 				if ((presv->ri_brp != NULL) &&
 					(presv->ri_wattr[RESV_ATR_interactive].at_flags &
 					ATR_VFLAG_SET)) {
-					presv->ri_wattr[RESV_ATR_interactive].at_flags &= ~ATR_VFLAG_SET;
-					snprintf(buf, sizeof(buf), "%s DENIED",
-						presv->ri_qs.ri_resvID);
-					(void)reply_text(presv->ri_brp,
-						PBSE_NONE, buf);
-					presv->ri_brp = NULL;
+					if (!(presv->ri_alter.ra_flags & RESV_ALTER_FORCED)) {
+						presv->ri_wattr[RESV_ATR_interactive].at_flags &= ~ATR_VFLAG_SET;
+						snprintf(buf, sizeof(buf), "%s DENIED",
```

Comment:
Since you're here, can you put these on one line? instead of putting them on multiple.